packages/components/src/assets/CircleWavyWarning.tsx

```diff
@@ -0,0 +1,35 @@
+import type { FC } from 'react'
+import React from 'react'
+
+// Icon from Phosphor Icons
+const CircleWavyWarning: FC = () => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    fill="currentColor"
+    viewBox="0 0 256 256"
+    width={30}
+    height={30}
+  >
+    <rect width="256" height="256" fill="none"></rect>
+    <path
+      d="M54.46089,201.53911c-9.204-9.204-3.09935-28.52745-7.78412-39.85C41.82037,149.95168,24,140.50492,24,127.99963,24,115.4945,41.82047,106.048,46.67683,94.31079c4.68477-11.32253-1.41993-30.6459,7.78406-39.8499s28.52746-3.09935,39.85-7.78412C106.04832,41.82037,115.49508,24,128.00037,24c12.50513,0,21.95163,17.82047,33.68884,22.67683,11.32253,4.68477,30.6459-1.41993,39.8499,7.78406s3.09935,28.52746,7.78412,39.85C214.17963,106.04832,232,115.49508,232,128.00037c0,12.50513-17.82047,21.95163-22.67683,33.68884-4.68477,11.32253,1.41993,30.6459-7.78406,39.8499s-28.52745,3.09935-39.85,7.78412C149.95168,214.17963,140.50492,232,127.99963,232c-12.50513,0-21.95163-17.82047-33.68884-22.67683C82.98826,204.6384,63.66489,210.7431,54.46089,201.53911Z"
+      fill="none"
+      stroke="currentColor"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    ></path>
+    <line
+      x1="127.99951"
+      y1="80"
+      x2="127.99951"
+      y2="136"
+      fill="none"
+      stroke="currentColor"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    ></line>
+    <circle cx="127.99951" cy="172" r="8"></circle>
```

Comment:
It seems to have a bug here. Try to add the `currentColor` on this `<circle>` too.
![image](https://user-images.githubusercontent.com/11613011/225989621-dcc6253b-be65-4ae6-8450-8edff8fdefd5.png)

Reply:
👀  I replaced the Icons for the updated ones from Phosphor Icons, could you double check? 
|Before|After|
|-|-|
|<img width="286" alt="Screenshot 2023-03-20 at 16 10 03" src="https://user-images.githubusercontent.com/11325562/226459299-975dedd8-35a4-4a73-9c8e-585d4cf8372c.png">|<img width="426" alt="Screenshot 2023-03-20 at 17 32 34" src="https://user-images.githubusercontent.com/11325562/226459337-c4252afe-4730-4b43-a05e-106749ea301d.png">|
|<img width="407" alt="Screenshot 2023-03-20 at 17 33 42" src="https://user-images.githubusercontent.com/11325562/226459546-890326f8-d8f8-403e-ad41-c911a8370074.png">|<img width="417" alt="Screenshot 2023-03-20 at 17 32 27" src="https://user-images.githubusercontent.com/11325562/226459323-85839872-e7ef-4a85-8fcc-da2f566c09c5.png">|